### ISSUE TITLE
Build Fails

### ISSUE THREAD

**Issue by elmcapp** [orchetect/MIDIKit#91]
I have a React Native project and trying to use this package to create a native midi module. I am having a problem that once I import this Package the build will fail. It fails even without importing and using it in any of my source code. Simple adding the package to the project causes my build to fail. I am not sure if there are configurations that are require or should be avoided.

I attached my Xcode logs
[Build target test_react-macOS_2022-06-03T02-11-57.txt](https://github.com/orchetect/MIDIKit/files/8830045/Build.target.test_react-macOS_2022-06-03T02-11-57.txt)

<img width="850" alt="Screen Shot 2022-06-03 at 2 25 03 AM" src="https://user-images.githubusercontent.com/64275658/171798897-9ba25b1f-9d67-45e5-a59e-f6061fbe3d1c.png">

**Comment by orchetect** [comment]
I don't have experience with React Native but from Googling, it seems that it's possible to use Swift Packages in a RN application -- but there may be some boilerplate required. I won't have a chance to look deeper until next week.

Not sure if any of these help:

- https://stackoverflow.com/questions/65130609/how-to-solve-undefined-symbols-for-architecture-x86-64-on-nsscanner-swift-pod
- https://cuneyt.aliustaoglu.biz/en/using-native-ui-libraries-in-react-native-for-with-swift/

**Comment by elmcapp** [comment]
Found a working solution from the first link that you provided. Looks like there may be a conflict between React Native when MidiKit is added. Would be nice if there is a simple fix as I don't know what other issues may occur  in React Native after making the changes below. The working solution is:
- The solution: Remove the swift-5.0 search path, or you will get an error about undefined symbols (try it if you don't believe me), do this -> https://github.com/react-native-community/upgrade-support/issues/62#issuecomment-622985723

Basically for React Native users trying to use MidiKit you have to remove all search path for debug and release.
<img width="1400" alt="Screen Shot 2022-06-03 at 3 18 30 AM" src="https://user-images.githubusercontent.com/64275658/171807333-f95677a7-f8db-466d-9053-8cf2d54fd940.png">

**Comment by orchetect** [comment]
Thanks for confirming that. I'll include this tip in troubleshooting documentation. It may be a workaround but if it works then it's worth noting for the time being.
